civictechprojects/static/css/partials/_SubHeader.scss

```diff
@@ -11,22 +11,84 @@
   text-decoration: inherit;
   width: auto;
   cursor: pointer;
+  float: none;
 }
 
 .SubHeader-root {
   background-color: $color-background-dark;
-  height: 50px;
-  padding: 0 80px 0 80px;
+  height: 150px;
+  padding: 0 5% 0 5%;
+  margin-bottom: 5%;
 }
 
 .SubHeader-sectionButton {
   color: $color-text-light;
   display: inline-block;
   font-weight: bold;
-  margin-top: 18px;
+  margin: auto;
   padding-right: 40px;
 }
 
 .SubHeader-rightContent {
-  float: right;
+  display: flex; 
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
One quick thing I missed in my review, sorry! Could you set the justify-content so it only applies `center` on mobile? On desktop resolutions it should be the default `flex-start` - the nav menu looks odd for a logged out user otherwise.